### DISCUSSION TITLE
Question

### DISCUSSION THREAD

**Discussion by bshlomo** [twbs/bootstrap#32231] [Q&A] [answered]
Sorry if this is not the place for such questions, but I can't find any other place

Hi
Can I use Bootstrap with pure dart:html application?
If the answer is yes, please guide me to a link with some info

Thank you
Shlomo
shlomo.belleli@gmail.com

**Comment by techboyg5** [comment]
Bootstrap is a CSS stylesheet.

If you can use CSS, you can use Bootstrap.

**Reply by mdo** [reply]
This is not correct—Bootstrap provides HTML, CSS, and JS. And that's just in the distribution files.

**Comment by mdo** [comment, marked as answer]
Yes, you most likely can use Bootstrap with dart:html. We don't use Dart ourselves so you likely won't find something here for it, but a quick Google search shows there are some options out there.
